manifests/dashboard/maintenance.pp

```diff
@@ -33,17 +33,20 @@
 
   validate_absolute_path($reports_spool_dir)
   validate_absolute_path($dump_dir)
+  validate_string($dump_database_command)
 
-  if $reports_days_to_keep == '30' {
-    $my_purge_old_reports_command = $purge_old_reports_command
-  } else {
-    $my_purge_old_reports_command = "/usr/bin/rake -f /usr/share/puppet-dashboard/Rakefile RAILS_ENV=production reports:prune upto=${reports_days_to_keep} unit=day >> /var/log/puppet/dashboard_maintenance.log"
+  if is_integer($reports_days_to_keep) == false {
```

Comment:
you could use `validate_integer()` here since you are effectively just doing validation with this conditional